ui/apps/everest/src/pages/db-cluster-details/db-cluster-details.tsx

```diff
@@ -21,7 +21,6 @@ export const DbClusterDetails = () => {
     namespace,
     {
       enabled: !!namespace && !!dbClusterName,
-      refetchInterval: 5 * 1000,
```

Comment:
removed the interval from db-cluster-details root component, we need this only for overview tab